core/lb.c

```diff
@@ -929,6 +933,9 @@ void
 oio_lb_world__feed_slot (struct oio_lb_world_s *self, const char *name,
 		const struct oio_lb_item_s *item)
 {
+	if (!item || item->weight <= 0)
+		return;
```

Comment:
This is dangerous. This will make the service undiscoverable. You don't wan't undiscoverable, you just wan't unselectable. There is a check in `oio_lb_world__feed_slot_unlocked` for this condition, after the item has been integrated in the _world_.